manifests/0000_26_cloud-controller-manager-operator_01_images.configmap.yaml

```diff
@@ -17,5 +17,6 @@ data:
       "cloudControllerManagerGCP": "quay.io/openshift/origin-gcp-cloud-controller-manager",
       "cloudControllerManagerIBM": "quay.io/openshift/origin-ibm-cloud-controller-manager",
       "cloudControllerManagerOpenStack": "quay.io/openshift/origin-openstack-cloud-controller-manager",
-      "cloudControllerManagerVSphere": "quay.io/openshift/origin-vsphere-cloud-controller-manager"
+      "cloudControllerManagerVSphere": "quay.io/openshift/origin-vsphere-cloud-controller-manager",
+      "cloudControllerManagerPowerVS": "quay.io/openshift/origin-powervs-cloud-controller-manager"
```

Comment:
The rest of the providers in this list are alphabetical, can we move this to the correct place alphabetically

Reply:
Updated it in the latest commit